database/Scripts/Fill-unique-bathymetry-geoms-table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE [dbo].[UniqueBathymetryGeoms] (
+  [CATEGORY] VARCHAR(10) NOT NULL,
+  [RANK]     INT         NOT NULL,
+  [geom]     GEOMETRY    NOT NULL
+);
+
+INSERT INTO [dbo].[UniqueBathymetryGeoms] ([CATEGORY], [RANK], [geom])
+SELECT
+  [CATEGORY],
+  [RANK],
+  [geom].STDifference((
+    SELECT geometry::UnionAggregate([b2].[geom])
+    FROM [dbo].[BathymetryGeoms] AS [b2]
+    WHERE [b2].[RANK] < [b1].[RANK]
+  )) AS [geom]
+FROM [dbo].[BathymetryGeoms] AS [b1];
```

Comment:
Not too fussed at this instant, but that script (whose name implies it's filling something) also creates the thing it's filling -- which also of course means you can't run it twice.

(comment applies to a couple of others I think)